### PR TITLE
SM-861 - Device definitions sync tableland remove empty values

### PR DESCRIPTION
**Pull request by rhvivancoeffio**
do not have empty attributes

### CODE REVIEW

File: scripts/tableland/tasks.ts [hunk 1]
```diff
@@ -582,12 +589,21 @@ task('sync-tableland', 'npx hardhat sync-tableland --network <networkName>')
...
+                        if (different.length == 0) {
+                            const hasEmptyValues = dds[0].metadata.device_attributes.some(({ value }) => !value);
+                            if (hasEmptyValues) {
+                                updateDeviceDefinitionByManufacturers.push(element);
+                            }
+                        }
+
                     }
-                    
+
```
Comment (JamesReate) [inline scripts/tableland/tasks.ts:606]: nit: no need for empty line

File: scripts/tableland/tasks.ts [hunk 2]
```diff
@@ -617,12 +633,12 @@ task('sync-tableland', 'npx hardhat sync-tableland --network <networkName>')
                         transactionHash: (await tx.wait())?.hash as string,
                     });
                 }
-            
+
```
Comment (JamesReate) [inline scripts/tableland/tasks.ts:636]: nit: no need for empty line

**Review by JamesReate — APPROVED** [review]
lgtm - i don't see tests but seems normal here

**Comment by JamesReate** [comment]
@LorranSutter bumping this, see if code looks ok with you to merge

**Comment by elffjs** [comment]
Lorran is out until next week

**Review by LorranSutter — APPROVED** [review]

**Pull request merged**